beetsplug/replaygain.py

```diff
@@ -83,10 +88,108 @@ def compute_album_gain(self, album):
         raise NotImplementedError()
 
 
-# mpgain/aacgain CLI tool backend.
+# bsg1770gain backend
+
+
+class Bs1770gainBackend(Backend):
```

Comment:
Could you add a brief docstring here describing what "bs1770" is and roughly how it's used?